monit/README.md

```diff
@@ -0,0 +1,13 @@
+# Monit
+
+This plan packages [monit](https://mmonit.com/monit), a monitoring agent with powers (it can restart processes on failure and so on).
+
+## Usage
+
+When using monit, you'll want to add your own rules, this cannot be done with this package. You'll need to create your own which would depend on this one. The configuration is provided as a model for you to speed up your package development.
+
+## Plan supporter
+
+Romain Sertelon @rsertelon
```

Comment:
Want to add yourself to the [CODEOWNERS](https://github.com/habitat-sh/core-plans/blob/master/CODEOWNERS) for this plan?

Reply:
I could yes, what are the implications exactly? :)

Reply:
@elliott-davis ? ;)

Reply:
Done ;) and for other packages too :)